model/bucket_test.go

```diff
@@ -0,0 +1,133 @@
+package model
```

Comment:
There are too many tests for such limited functionality. Check the status of the existing tests.